core/src/test/java/com/sequenceiq/cloudbreak/core/flow2/cluster/provision/service/ClusterProxyServiceTest.java

```diff
@@ -301,25 +304,12 @@ void testRegisterGatewayConfigurationWithoutCcmEnabled() {
         verify(clusterProxyRegistrationClient).updateConfig(captor.capture());
         ConfigUpdateRequest gatewayUpdateRequest = captor.getValue();
         assertThat(gatewayUpdateRequest.getClusterCrn()).isEqualTo(STACK_CRN);
-        assertEquals("https://1.2.3.4/test-cluster", gatewayUpdateRequest.getUriOfKnox(), "Gateway Knox URI should match");
-    }
-
-    @Test
-    void testRegisterGatewayConfigurationWithCcmV1Enabled() {
-        when(stackService.getByIdWithListsInTransaction(STACK_ID)).thenReturn(testStackUsingCCMAndKnox());
-
-        underTest.registerGatewayConfiguration(STACK_ID);
-
-        ArgumentCaptor<ConfigUpdateRequest> captor = ArgumentCaptor.forClass(ConfigUpdateRequest.class);
-        verify(clusterProxyRegistrationClient).updateConfig(captor.capture());
-        ConfigUpdateRequest gatewayUpdateRequest = captor.getValue();
-        assertThat(gatewayUpdateRequest.getClusterCrn()).isEqualTo(STACK_CRN);
-        assertEquals("https://10.10.10.10/test-cluster", gatewayUpdateRequest.getUriOfKnox(), "CCMV1 Knox URI should match");
+        assertEquals("https://1.2.3.4:9443/knox/test-cluster", gatewayUpdateRequest.getUriOfKnox(), "Gateway Knox URI should match");
     }
 
     @ParameterizedTest
-    @EnumSource(value = Tunnel.class, names = {"CCMV2", "CCMV2_JUMPGATE"}, mode = EnumSource.Mode.INCLUDE)
-    void testRegisterGatewayConfigurationWithCcmV2Enabled(Tunnel tunnel) {
+    @EnumSource(value = Tunnel.class, names = {"CCMV2", "CCMV2_JUMPGATE", "CCM"}, mode = EnumSource.Mode.INCLUDE)
+    void testRegisterGatewayConfigurationWithCcmV2Enabled(Tunnel tunnel) throws JsonProcessingException {
```

Comment:
> Some general comments for the PR:
> 
> * As I see, there are two commits, one for CB-14473 and another for CB-14474. If you think it indeed makes sense to bundle these together (e.g. because re-registration is not meaningful / effective without the URL changes of the other commit), please update the PR title to reflect that it is a joint change for those 2 tickets.
> * The sentence "... instead of that use the CCMv2 approach which" in the PR description seems to be incomplete.

Updated the description and title, the first effort is a joint effort with @doktoric and it could be a separate commit, I prefer that.